### PR TITLE
PR: Add cards directive

### PR DESCRIPTION
**Pull request by dalthviz**
### Pull Request Checklist

* [x] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [x] Based your PR on the latest version of the correct branch (master or 3.x)
* [x] Checked your writing carefully for correct English spelling, grammar, etc
* [x] Described your changes and the motivation for them below
* [ ] Noted what issue(s) this pull request resolves, creating one if needed

## Description of Changes

A preview:

### 375 x 667

image

### 1366 x 768

image

### 1920 x 1080

image

### Issue(s) Resolved

Fixes #

### CODE REVIEW

**Comment by ccordoba12** [comment]
This looks awesome! Thanks @dalthviz for your work on it!

File: doc/conf.py [hunk 1]
```diff
@@ -42,8 +42,11 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
     "sphinx.ext.githubpages",
+    "sphinx_panels"
```
Comment (CAM-Gerlach) [inline doc/conf.py:45]:
```suggestion
    "sphinx_panels",
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -76,13 +76,15 @@ you can install it into your present environemnt with
...
 conda install sphinx
+pip install sphinx-panels
 pip install git+https://github.com/spyder-ide/spyder-docs-sphinx-theme.git@develop_spyder
 ```
 
 Or, if using ``pip``, you can grab it with
 
 ```bash
 pip install sphinx
+pip install sphinx-panels
```
Comment (CAM-Gerlach) [inline CONTRIBUTING.md:87]:
```suggestion
pip install sphinx sphinx-panels
```

File: .travis.yml [hunk 3]
```diff
@@ -14,6 +14,7 @@ env:
 install:
   - pip install -U pip
   - pip install sphinx doctr
+  - pip install sphinx-panels
```
Comment (CAM-Gerlach) [inline .travis.yml:17]:
```suggestion
  - pip install sphinx doctr sphinx-panels 
```

File: doc/index.rst [hunk 4]
```diff
@@ -9,38 +9,64 @@ Spyder Documentation
 Spyder is a powerful scientific environment written in Python, for Python, and designed by and for scientists, engineers and data analysts.
 It features a unique combination of the advanced editing, analysis, debugging, and profiling functionality of a comprehensive development tool with the data exploration, interactive execution, deep inspection, and beautiful visualization capabilities of a scientific package.
```
Comment (CAM-Gerlach) [inline doc/index.rst:11]:
```suggestion



```

The line breaks were removed here for some reason.

**Comment by ccordoba12** [comment]
@dalthviz, could you also update the screenshots with the previous color for icons? Just to check how everything is going to look.

Other than that, I have no more comments here. I'll leave the final approval to @juanis2112.

**Pull request merged**